### ISSUE TITLE
Android SDK Platform 29?

### ISSUE THREAD

**Issue by neoncube2** [godotengine/godot#63878]
### Godot version

3.4.5-stable

### System information

Windows 11

### Issue description

On https://docs.godotengine.org/en/stable/tutorials/export/exporting_for_android.html#doc-exporting-for-android, it says that for Android plugins, we should install Android SDK Platform 29.

Now that Godot 3.4.5 targets Android 31, should the documentation be updated to say Android SDK Platform 31, or is this not necessary?

Thanks 🙂  (Sorry, still trying to get my plugin working)

### Steps to reproduce

See above.

**Comment by akien-mga** [comment]
Yes this should be updated with the relevant values for `3.4`, `3.5` and `latest`. Moving to godot-docs repo.

It probably doesn't matter too much if you installed different versions though, I think gradle will automatically install the versions it needs if missing. CC @m4gr3d
